frontend/src/SignInUpModal.js

```diff
@@ -0,0 +1,251 @@
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showModal: false,
+      loggedIn: false,
+      loading: false,
+      error: null,
+      initialTab: null,
+      recoverPasswordSuccess: null,
+    };
+  }
+
+  onLogin() {
+    console.log('__onLogin__');
+    console.log('email: ' + document.querySelector('#email').value);
+    console.log('password: ' + document.querySelector('#password').value);
+
+    const email = document.querySelector('#email').value;
+    const password = document.querySelector('#password').value;
+
+    if (!email || !password) {
+      this.setState({
+        error: true
+      })
+    } else {
+      this.onLoginSuccess('form');
+    }
+  }
+
+  onRegister() {
+    console.log('__onRegister__');
+    console.log('login: ' + document.querySelector('#login').value);
+    console.log('email: ' + document.querySelector('#email').value);
+    console.log('password: ' + document.querySelector('#password').value);
+
+    const login = document.querySelector('#login').value;
+    const email = document.querySelector('#email').value;
+    const password = document.querySelector('#password').value;
+
+    if (!login || !email || !password) {
+      this.setState({
+        error: true
+      })
+    } else {
+      this.onLoginSuccess('form');
+    }
+  }
+
+  onRecoverPassword() {
+    console.log('__onFotgottenPassword__');
+    console.log('email: ' + document.querySelector('#email').value);
+
+    const email = document.querySelector('#email').value;
+
+    if (!email) {
+      this.setState({
+        error: true,
+        recoverPasswordSuccess: false
+      })
+    } else {
+      this.setState({
+        error: null,
+        recoverPasswordSuccess: true
+      });
+    }
+  }
+
+  openModal(initialTab) {
+    this.setState({
+      initialTab: initialTab
+    }, () => {
+      this.setState({
+        showModal: true,
+      })
+    });
+  }
+
+  onLoginSuccess(method, response) {
+    this.closeModal();
+    this.setState({
+      loggedIn: method,
+      loading: false
+    })
+  }
+
+  onLoginFail(method, response) {
+    this.setState({
+      loading: false,
+      error: response
+    })
+  }
+
+  startLoading() {
+    this.setState({
+      loading: true
+    })
+  }
+
+  finishLoading() {
+    this.setState({
+      loading: false
+    })
+  }
+
+  afterTabsChange() {
+    this.setState({
+      error: null,
+      recoverPasswordSuccess: false,
+    });
+  }
+
+  closeModal() {
+    this.setState({
+      showModal: false,
+      error: null
+    });
+  }
+
+  render() {
+    const loggedIn = this.state.loggedIn
+      ? <div>
+          <p>You are signed in with: {this.state.loggedIn}</p>
+        </div>
+      : <div>
+          <p>You are signed out</p>
+      </div>;
+
+    const isLoading = this.state.loading;
+
+    return (
+      <div>
+
+        <button
+          className="RML-btn"
+          onClick={() => this.openModal('login')}
+        >
```

Comment:
I think for closing `>` it should be on the end of the `onClick` line. like 
![Screenshot 2020-06-25 at 2 08 11 PM](https://user-images.githubusercontent.com/26202764/85774953-52bf6200-b6ed-11ea-8d03-e0b3d7cf1bc5.png)

See here: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping